src/styles/home_nav.css

```diff
@@ -0,0 +1,225 @@
+.home-nav {
+  z-index: 1;
+  overflow: hidden;
+}
+.home-nav a {
+  text-decoration: none;
+  letter-spacing: 0.15em;
+  text-transform: uppercase;
+}
+
+.home-nav-menu a {
+  transition: color 0.3s ease;
+  color: #9da0a7;
+  font-size: .8em;
+  font-weight: 400;
+}
+
+.home-nav a:hover {
+  cursor: pointer;
+  color: #fff;
+}
+
+.nav-container {
+  padding-left: 40px;
+  padding-right: 40px;
+  background-color: #3d4351;
+  font-family: 'Catamaran', sans-serif;
+  font-weight: 300;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  overflow: hidden;
+}
+
+.logo{
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  margin: 14px 0 14px 0;
+  width: 210px;
+  height: 36px;
+}
+
+.logo-green-square {
+  background-color: #8dc63f;
+  height: 33px;
+  width: 33px;
+  min-width: 33px;
+  display: flex;
+  align-items: center;
+}
+
+.logo-green-square div {
+  font-size: 2em;
+  color: #3d4351;
+  font-weight: 200;
+  position:absolute;
+  left:46px;
+  top:9px;
+}
+
+.logo img {
+  max-height: 35px;
+}
+
+@media only screen and (max-width: 1200px) {
+  .logo-type {
+    visibility: hidden;
+  }
+}
+
+@media only screen and (max-width: 1030px) {
+  .logo-type {
+    visibility: visible;
+  }
+}
+
+.logo-type {
+  margin-left: 10px;
+  max-height: 15px !important;
```

Comment:
If we're using `!important`, we're doing something wrong...  Why is this needed here?